### ISSUE TITLE
extern on static and unused - inside device-save.c

### ISSUE THREAD

**Issue by miles992** [AFLplusplus/qemu-libafl-bridge#35]
I think there is something weird in `libafl_extras/syx-snapshot/device-save.c`. This line asks for an `extern` on a now `static` function:

line 16:
```c
extern void save_section_header(QEMUFile *f, SaveStateEntry *se, uint8_t section_type);
```

The only use of `save_section_header()` was previously inside `device_save_all()` and now moved mostly to `device_save_kind()`. In the new code `save_section_header()` is no longer used.

**Comment by miles992** [comment]
`save_section_header()` does the same, I guess it's a left over from the snapshot fix commit `bd1f40e27ee2360b477bfd63969363100dd8cb97`.

**Comment by rmalmain** [comment]
Indeed, thank you for your report. It should be fixed in commit 2967fb2580573bf4674babf4135fd12082049656.
